analysis/genie/variant_analysis/genie_search_analysis.ipynb

```diff
@@ -1228,43 +1254,48 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 47,
+   "execution_count": 48,
    "id": "efc9e0ad",
    "metadata": {},
    "outputs": [],
    "source": [
     "# Helper function\n",
     "def compute_percent(n: int, d: int) -> str:\n",
     "    \"\"\"Format percentage\n",
-    "    :param n, d: Two integers\n",
+    "    :param n: An integer\n",
+    "    :param d: An integer\n",
     "    :return: A string reporting the percentage\"\"\"\n",
     "    percent = round(100 * n / d, 1)\n",
-    "    return f\"{n} / {d} ({percent}%)\"\n",
+    "    return percent\n",
     "\n",
     "\n",
     "# Table 2 - Patient Matching. The percentages are computed using the total number of patients in the GENIE dataset.\n",
-    "summary_df_patient = pd.DataFrame(columns=[\"Protein\", \"Genomic\"])\n",
-    "summary_df_patient.loc[\"(%) of Patients with Normalized Variants Across Sources\"] = [\n",
+    "summary_df_patient = pd.DataFrame(columns=[\"Protein\", \"Genomic\", \"Protein or Genomic\"])\n",
```

Comment:
This section always uses `total_patients` in the `compute_percent` function. Just use that value in the function if it's not changing. Function parameters are good for values that are changing between calls

Reply:
@jarbesfeld I didn't mean hard code the value, I meant remove the parameter in the function and just use the `total_patients` variable there